src/test/java/wbooks/register/register.feature

```diff
@@ -0,0 +1,82 @@
+Feature: enter the required fields to create a new user
+
+  Background:
+    * url 'https://wbooks-api-stage.herokuapp.com/api/v1'
+
+    * def now = function(){ return java.lang.System.currentTimeMillis() }
+    * def email = 'email' + now() + '@wolox.test.com'
```

Comment:
This is a good idea, but it would be better to think of a function that adds up to one in each pass, but is not a for.
That´s great anyway

Reply:
Maybe I could use a function to try JS or Java. But I think this one is very simple, clean and effective so probably not need to change it 👀